### PR TITLE
updating the readme to reflect building docs

### PR DESCRIPTION
**Pull request by sguptatw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,29 +12,30 @@ The documents are written in RST and are built using [Sphinx](http://www.sphinx-
 - Every build will generate docs for all versions maintained.
 - The root documentation points to the latest released verion. Navigate to `master` version for the development version of this documentation.
 
+#### Pre-requisites
+
+* Python 3
+* Nodejs
+* NPM
+* pip3 install -U Sphinx
```
Comment (BugDiver) [inline README.md:20]: AFAIK NodeJs is not required. `pip` and  `Make` is required.
Also `pip3 install -U Sphinx` is not required, in fact, what required is `pip3 install -r requirements.txt` before running `make`, `sphinx` will be installed as part of that.

File: README.md [hunk 2]
```diff
@@ -12,29 +12,30 @@ The documents are written in RST and are built using [Sphinx](http://www.sphinx-
...
 
 ## Building the documentation
 
 There are two formats of documentation built for every version.
+- `pip3 install -r requirements.txt` to install the pre-requisites for docs.gauge.org
 
-- `make zip` will build `html` and zipped `singlehtml` documentation. The output is available in `_build/html` directory.
+- `make local-build` will build `html` and zipped `singlehtml` documentation. The output is available in `_build` directory.
```
Comment (BugDiver) [inline README.md:27]: I do not think any zip (`singlehtml`) is created, please verify that.

**Review by BugDiver — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sguptatw) [reply README.md:20]: Will remove pip3 install -U Sphinx. As far as NodeJS was concerned, I have only retained it. This was there earlier. I have just moved the pre-requisites before building the document.

File: README.md (hunk 2 above)
Comment (sguptatw) [reply README.md:27]: Yes, singlehtml is no longer created, removing that.

**Review by BugDiver — APPROVED** [review]

**Pull request merged**